src/map/clif.cpp

```diff
@@ -5242,6 +5242,13 @@ int clif_damage(struct block_list* src, struct block_list* dst, t_tick tick, int
 		}
 	}
 
+	// Calculate what sdelay to send to the client so it applies damage at the same time as the server
+	if (src->type == BL_MOB && battle_config.synchronize_damage) {
+		mob_data* md = BL_CAST(BL_MOB, src);
+		// The client only accepts values between 0 and 432; 432 stands for 1x animation speed
+		sdelay = sdelay * 432 / md->status.clientamotion;
```

Comment:
```suggestion
		if (md != nullptr && md->status.clientamotion != 0)
			sdelay = sdelay * 432 / md->status.clientamotion;
```

Reply:
Here it cannot be null, as "src" was already accessed before and "type" was already verified